lib/pages/events_page.dart

```diff
@@ -23,6 +24,24 @@ const List<BottomNavigationBarItem> bottomNavigationBarItems = [
   ),
 ];
 
+final Widget emptyListView = Container(
```

Comment:
This could be refactored into a separate file imo. Something like `widgets/NoEventsOverlay.dart`

Reply:
Done